src/component/menu/ApiDetails/ApiDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+
+interface PartialUiConfig {
```

Comment:
Previously in the frontend we've tried to follow a convention where we prepend each interface with an I so that we can easily recognize interfaces vs types. It's just a nice mental shortcut that helps distinguish an interface from a type in the code. `IPartialUiConfig` or something similar would follow that convention.